lib/core/test/manual/Cardano/Launcher/POSIXSpec.md

```diff
@@ -50,14 +50,14 @@ signals are correctly handled on a unix system.
 
 ### `SIGTERM`
 
-- Start the launcher in background using `stack exec -- cardano-wallet launch &`
+- Start the launcher in background using `stack exec -- cardano-wallet-jormungandr launch --genesis-block lib/jormungandr/test/data/jormungandr/block0.bin --bft-leaders lib/jormungandr/test/data/jormungandr/secret.yaml &`
```

Comment:
`--bft-leaders` doesn't exist anymore. This should be updated to reflect the latest version :+1: